googler

```diff
@@ -19,16 +19,21 @@
 
 from __future__ import print_function
 import argparse
+import atexit
 import collections
+import functools
 import gzip
 import http.client
 import html.entities
 import html.parser
 import io
+import json
+import logging
```

Comment:
Logging is always required, because errors and warnings are emitted through the logger.